css/build.css

```diff
@@ -4,7 +4,7 @@ img {
 }
 
 img[data-image-id] {
-  max-width: 100%;
+  max-width: 92vw;
```

Comment:
1. Please explain the fix
2. Why 92?
3. Why does the linked PR have 94 on the image editor?

@tonytlwu cc you may want to review as well

Reply:
@squallstar 
1. If we use % IE 11 shows an image full-sized like this:
![image](https://user-images.githubusercontent.com/53430352/71360095-52f07080-2597-11ea-902a-e1f8bce18164.png)
With the `vw`, this is not happening. The thing is that if I drop my component it happens, but if I drop the current master component it works fine but there is no difference between them.
2. If we give an image more than this size it will go beyond view borders.
3. Same reason as 2 but only in linked PR 94 looks better and do not go beyond view borders

@tonytlwu @squallstar  in light of what I said in the 1. Maybe it would be better to close this PR?

Reply:
Let's close this.